erpnext/setup/doctype/item_group/item_group.py

```diff
@@ -43,24 +43,27 @@ def on_update(self):
 
 	def make_route(self):
 		'''Make website route'''
-		if not self.route:
-			self.route = ''
-			if self.parent_item_group:
-				parent_item_group = frappe.get_doc('Item Group', self.parent_item_group)
+		self.route = ''
+		if self.parent_item_group:
+			parent_item_group = frappe.get_doc('Item Group', self.parent_item_group)
```

Comment:
Why get_doc? you have to get only 2 field use 
frappe.db.get_value("Item Group", self.parent_item_group, ["parent_item_group", "route"])

Reply:
Fixed